ESPHome/TX-Ultimate-Easy-ESPHome_core_hw_speaker.yaml

```diff
@@ -17,8 +17,8 @@ i2s_audio:
   i2s_lrclk_pin: GPIO4
 
 media_player:
-  - id: mp_speaker
-    name: Speaker
+  - id: mp_media_player
+    name: Media Player
```

Comment:
Nope!
The BLE addon have another component added with the name `mp_speaker` and removes the `media_player` component entirely.
I've changed the name on the media player exactly to avoid mistakes when using the speaker (as in BLE add-on).